docusaurus/docs/core-concepts/metrics/metrics.md

```diff
@@ -2,7 +2,7 @@
 
-[![](https://mermaid.ink/img/pako:eNqdlcFP5CAUxv-VF05rMjuJc2w2c1EPZuMedI8kBtvnSKSPkb66zhr_d6FjR6BUo-2JDz4--D1ankVtGxSV6PChR6rxVKuNU60k1mwQLpCdrjtJkrbKsa71VhHDOWnWijVt4Or0d9p31mwQLm3P6OA47fqD_M-6ezixxM4ag27eukq7_qJrNUWR45suBX6u18kCoALf5bDrwD3tHUlKMGRzQ4V7Aw8GsoxgH_3gNGoBM0FL97S82TF2TjH-unHrSG-7TVnt9H-EIMeB2cLSxFWUyDOJXEzkPDFlchyYZJt9pxig5MAy6GFlGDP_wo6wjBBLBPFTgJ9VDMv4sEQPZ-D5UwqSwD_T4106jpIUNW-OCfbCFNXYgJM7RYQGftTszJEkpOZr8asP4lffi0_Lm-xnAcXpgntpPFSqd4Hg0PY_nx6vWbd4kPIjMIpRSSIpFOQgccnKU-t7LWf-DNPaTSqW8zWa7j-EkhnC-BjH0E5xDFKOYxSjPUXSiGOQuGTlqTXGIRai9d-q0o2_HJ7DcZGC77BFKTwG0eCt6g1LIenFD1U926sd1aJi1-NC9NvGT_x2l4jqVpnuoJ41mq07iDg0L_a30HAZvbwCsdRbfQ?type=png)](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNqdlcFP5CAUxv-VF05rMjuJc2w2c1EPZuMedI8kBtvnSKSPkb66zhr_d6FjR6BUo-2JDz4--D1ankVtGxSV6PChR6rxVKuNU60k1mwQLpCdrjtJkrbKsa71VhHDOWnWijVt4Or0d9p31mwQLm3P6OA47fqD_M-6ezixxM4ag27eukq7_qJrNUWR45suBX6u18kCoALf5bDrwD3tHUlKMGRzQ4V7Aw8GsoxgH_3gNGoBM0FL97S82TF2TjH-unHrSG-7TVnt9H-EIMeB2cLSxFWUyDOJXEzkPDFlchyYZJt9pxig5MAy6GFlGDP_wo6wjBBLBPFTgJ9VDMv4sEQPZ-D5UwqSwD_T4106jpIUNW-OCfbCFNXYgJM7RYQGftTszJEkpOZr8asP4lffi0_Lm-xnAcXpgntpPFSqd4Hg0PY_nx6vWbd4kPIjMIpRSSIpFOQgccnKU-t7LWf-DNPaTSqW8zWa7j-EkhnC-BjH0E5xDFKOYxSjPUXSiGOQuGTlqTXGIRai9d-q0o2_HJ7DcZGC77BFKTwG0eCt6g1LIenFD1U926sd1aJi1-NC9NvGT_x2l4jqVpnuoJ41mq07iDg0L_a30HAZvbwCsdRbfQ)
+[![](https://mermaid.ink/img/pako:eNqdlUFP4zAQhf_KyKdF6laixwj1AhzQij0sHC2tTDIUC2dcnMlCQfz3tdOmtR23WrY5-TnPz_ONG3-I2jYoKtHhS49U45VWK6daSazZINwiO113kiStlWNd67UihhvSrBVrWsHd1Y907rpZIfyyPaOD83TqJ_Krdc9waYmdNQbdcesinbpH12qKIscn3Qp8Xy6TDUAFfsph14F72zqSlGDI1oYKtwYeDGQZwf7xL6dRMzgSNHdv84cNY-cU48WDW0Z6263KaqffEYIcB2YbSxMXPnG70TlngQc5ykvEKC4Fch6AZJUeEAYiOa2M-GFbO-BfL8eVy3Glck7Q-9d28ZF2cbFdE37-lIIk8L_p8S4dR0mKmp1jQr6wRDUO4PJJEaGBbzU7cyYJqfla_OJE_OL_4tMOJ_XMoLhccM-Np0r1JhAcxv7j0-Nv1i3upfwUjGLUk0gKDdlLXLLy1Hro5ZEvw7R3k47lfI2m55NQMkN4P8YxjFMcg5TjGMWopkgacQwSl6w8tcY4xEy0_u-qdOMvh49wXKTgJ2xRCo9BNPioesNSSPr0r6qe7d2GalGx63Em-nXjF97dJaJ6VKbbq9eNZuv2Ig7D2-0tNFxGn38BXaRbfQ?type=png)](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNqdlUFP4zAQhf_KyKdF6laixwj1AhzQij0sHC2tTDIUC2dcnMlCQfz3tdOmtR23WrY5-TnPz_ONG3-I2jYoKtHhS49U45VWK6daSazZINwiO113kiStlWNd67UihhvSrBVrWsHd1Y907rpZIfyyPaOD83TqJ_Krdc9waYmdNQbdcesinbpH12qKIscn3Qp8Xy6TDUAFfsph14F72zqSlGDI1oYKtwYeDGQZwf7xL6dRMzgSNHdv84cNY-cU48WDW0Z6263KaqffEYIcB2YbSxMXPnG70TlngQc5ykvEKC4Fch6AZJUeEAYiOa2M-GFbO-BfL8eVy3Glck7Q-9d28ZF2cbFdE37-lIIk8L_p8S4dR0mKmp1jQr6wRDUO4PJJEaGBbzU7cyYJqfla_OJE_OL_4tMOJ_XMoLhccM-Np0r1JhAcxv7j0-Nv1i3upfwUjGLUk0gKDdlLXLLy1Hro5ZEvw7R3k47lfI2m55NQMkN4P8YxjFMcg5TjGMWopkgacQwSl6w8tcY4xEy0_u-qdOMvh49wXKTgJ2xRCo9BNPioesNSSPr0r6qe7d2GalGx63Em-nXjF97dJaJ6VKbbq9eNZuv2Ig7D2-0tNFxGn38BXaRbfQ)
```

Comment:
Alternatively, consider embedding the Mermaid diagram in the Markdown like this.

    ```mermaid
    sequenceDiagram
    ...
    ```